src/microbe_stage/OrganelleDefinition.cs

```diff
@@ -276,31 +276,31 @@ public enum OrganelleGroup
     ///   boxing)
     /// </para>
     /// <returns>True when this has a scene</returns>
-    public bool TryGetGraphicsScene(out LoadedSceneWithModelInfo modelInfo)
+    public bool TryGetGraphicsScene(OrganelleUpgrades? upgrades, out LoadedSceneWithModelInfo modelInfo)
```

Comment:
XML doc is missing documentation for the upgrades parameter.

Reply:
The XML documentation should be something like `Applied upgrades for this organelle. This is required as some upgrades alter organelle visuals`. Also XML elements that fit on a single line should be placed on a single line.